lib/config.js

```diff
@@ -36,6 +37,18 @@ class Config {
     this.tryOrWarn(() => {
       this.features = loadFeatures(this.paths().features);
     });
+    if (this.filters.length) {
+      this.filterFeatures();
+    }
+  }
+
+  filterFeatures() {
+    let filters = this.filters.map((path) => {
+      return this.base + '/' + path;
```

Comment:
Probably! Going to worry about this later. Also, I think you may need to add me as an owner so I can push the new stuff to npm.

Using the per file running of cucaroo has become super useful.